src/main/java/org/schemaspy/output/diagram/graphviz/GraphvizDot.java

```diff
@@ -75,7 +75,7 @@ private GraphvizVersion initVersion() {
             LOGGER.trace("GraphvizVersion: \"{}\"", versionLine);
 
             // look for a number followed numbers or dots
-            Matcher matcher = Pattern.compile("[0-9]+\\.[0-9]+[^\\.]").matcher(versionLine);
+            Matcher matcher = Pattern.compile("([0-9]+\\.)+[0-9]+[^.]").matcher(versionLine);
```

Comment:
I don't know why I made it so complicated and at the same time incorrect.

We can simplify it to `\\d+\\.\\d+\\.\\d+`

The negative match was to only get major.minor, but I don't think that it's of interest anymore.

Reply:
I think this suggestion is incorrect based on the comment above; it looks like sometimes there's two-component versions. Definitely I can replace the [0-9] with \d though. Agreed, greedy matching probably also makes the negative match unnecessary: my regex will just eat the entire version. As far as I can tell, the downstream code can deal with unequal length versions and just look at the beginning so this should be fine.

Reply:
> The negative match was to only get major.minor, but I don't think that it's of interest anymore.

I've proposed to remove it all together since the version is only used to check 5 year old version numbers.

To get only major.minor it would be `\\d+\\.\\d+` which would be simpler than the negative matcher and capturing groups.